pkg/kubelet/status_manager.go

```diff
@@ -104,7 +104,22 @@ func (s *statusManager) SyncBatch() {
 			podFullName := kubecontainer.GetPodFullName(pod)
 			status := syncRequest.status
 			glog.V(3).Infof("Syncing status for %s", podFullName)
-			_, err := s.kubeClient.Pods(pod.Namespace).UpdateStatus(pod.Name, &status)
+
+			var err error
+			statusPod := &api.Pod{
```

Comment:
statusPod is always overwritten in line 115, what's the point of defining it here?